### ISSUE TITLE
C# Interactive: How to set PrintOptions.MemberDisplayFormat

### ISSUE THREAD

**Issue by mazhou-msft** [dotnet/roslyn#33370]
**Version Used**: 
Visual Studio 2019 Preview 4 28612.98.d16.0

**Steps to Reproduce**:

1. Open C# Interactive
2. Type `PrintOptions.MemberDisplayFormat = `
3. The IntelliSense will auto select MemberDisplayFormat enum in the completion list, type a `.` to try to list the enum values.

**Expected Behavior**:
The values should be populated: `SingleLine`, `SeparateLines` and `Hidden`, and compiler should not report compilation error.

**Actual Behavior**:
1. No valid enum values are populated
2. Compiler reports `MemberDisplayFormat` type is not found in the current context
3. Cannot directly set this option, to change the member display format, you will need to do this:
```c#
> var t = Type.GetType("Microsoft.CodeAnalysis.Scripting.Hosting.MemberDisplayFormat, Microsoft.CodeAnalysis.Scripting");
> var t = Type.GetType("Microsoft.CodeAnalysis.Scripting.Hosting.MemberDisplayFormat, Microsoft.CodeAnalysis.Scripting");
> var e = Enum.Parse(t, "SeparateLines");
> PrintOptions.MemberDisplayFormat = (dynamic)e;
```
